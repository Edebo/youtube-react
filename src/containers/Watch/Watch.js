import React from "react";
import "./Watch.scss";
import { Video } from "../../components/Video/Video";
import { VideoMetadata } from "../../components/VideoMetadata/VideoMetadata";
import { RelatedVideos } from "../../components/RelatedVideos/RelatedVideos";
import { VideoInfoBox } from "../../components/VideoInfoBox/VideoInfoBox";
import { Comments } from "../Comments/Comments";
export class Watch extends React.Component {
  render() {
    return (
      <div style={{ maxWidth: "80%" }}>
        <Video className='video' id='-7fuHEEmEjs' />
        <VideoMetadata className='metadata' viewCount={1000} />
        <VideoInfoBox className='video-info-box' />
        <Comments className='comments' />
        <RelatedVideos className='related-videos' />
      </div>
    );
  }
}
