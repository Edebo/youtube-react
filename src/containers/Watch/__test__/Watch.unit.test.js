import React from "react";
import { shallow } from "enzyme";
import { Watch } from "../Watch";

describe("", () => {
  test("rendering the watchcomponent", () => {
    const wrapper = shallow(<Watch />);
    expect(wrapper).toMatchSnapshot();
  });
});
