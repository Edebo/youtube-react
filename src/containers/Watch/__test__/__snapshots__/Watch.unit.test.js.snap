// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering the watchcomponent 1`] = `
<div
  style={
    Object {
      "maxWidth": "80%",
    }
  }
>
  <Video
    className="video"
    id="-7fuHEEmEjs"
  />
  <VideoMetadata
    className="metadata"
    viewCount={1000}
  />
  <VideoInfoBox
    className="video-info-box"
  />
  <Comments
    className="comments"
  />
  <RelatedVideos
    className="related-videos"
  />
</div>
`;
